yarn-project/circuit-types/src/logs/l1_payload/l1_note_payload.ts

```diff
@@ -136,4 +147,26 @@ export class L1NotePayload extends L1Payload {
       this.noteTypeId.equals(other.noteTypeId)
     );
   }
+
+  static #getPublicValuesAndRemainingCipherText(input: Buffer): [Fr[], Buffer] {
```

Comment:
It'd be good to have some description of the content layout here, either text or a simple diagram

Reply:
https://github.com/AztecProtocol/aztec-packages/pull/8538/commits/f7b1ddcc5d0b3ea257bc9816e0f716efec13292b